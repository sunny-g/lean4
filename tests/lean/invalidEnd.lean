namespace Foo.Bar
section Baz
section

end Baz
end Bal
end Fool.Bar
end
