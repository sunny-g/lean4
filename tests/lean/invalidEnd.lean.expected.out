invalidEnd.lean:5:0-5:7: error: invalid 'end', name mismatch (expected nothing)
invalidEnd.lean:6:0-6:7: error: invalid 'end', name mismatch (expected Baz)
invalidEnd.lean:7:0-7:12: error: invalid 'end', name mismatch (expected Foo.Bar)
invalidEnd.lean:8:0-8:3: error: invalid 'end', insufficient scopes
